src/components/CreatureDetail.js

```diff
@@ -11,6 +12,7 @@ class CreatureDetail extends Component {
       creatureType: '',
       creatureMove: '',
       creatureActions: [],
+      isFav: false,
```

Comment:
I can remove this isFav, as I didn't end up using that as a reference anywhere else.